main.tf

```diff
@@ -49,17 +74,17 @@ locals {
     "essential": true,
     "portMappings": [
       {
-        "containerPort": 80,
-        "hostPort": 80,
-        "protocol": "tcp"
+	"containerPort": 80,
+	"hostPort": 80,
+	"protocol": "tcp"
       }
     ],
     "logConfiguration": {
       "logDriver": "awslogs",
       "options": {
-        "awslogs-group": "${local.awslogs_group}",
-        "awslogs-region": "${data.aws_region.current.name}",
-        "awslogs-stream-prefix": "nginx"
+	"awslogs-group": "${local.awslogs_group}",
+	"awslogs-region": "${data.aws_region.current.name}",
+	"awslogs-stream-prefix": "nginx"
```

Comment:
What's up with this weird whitespace adjustment?

Reply:
Some how tabs snuck in there